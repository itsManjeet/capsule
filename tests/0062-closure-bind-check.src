let Holder = fun (value) {
    let self = {};
    self.value = value;

    self.getValue = fun () {
        return self.value;
    };
    return self;
};

let h1 = Holder(12);
let h2 = Holder(139);

println (h1.value);
println (h2.value);

println (h1.getValue());
println (h2.getValue());