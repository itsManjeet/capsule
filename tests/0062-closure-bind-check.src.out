12
139
12
139
